modules/space/rotation/spicerotation.cpp

```diff
@@ -80,6 +109,15 @@ namespace {
         // [[codegen::verbatim(FixedDateInfo.description)]]
         std::optional<std::string> fixedDate
             [[codegen::annotation("A time to lock the rotation to")]];
+
+        // [[codegen::verbatim(UseLightTravelTimeInfo.description)]]
+        std::optional<bool> useLightTravelTime;
+
+        // [[codegen::verbatim(LightTravelTimeObserverInfo.description)]]
+        std::optional<std::string> lightTravelTimeObserver;
+
+        // [[codegen::verbatim(LightTravelTimeTargetInfo.description)]]
+        std::optional<std::string> lightTravelTimeTarget;
```

Comment:
To prevent new error conditions, I'd suggest instead:
```cpp
struct LightTravelTime {
  std::string observer;
  std::string target;
};

std::optional<LightTravelTime> lightTravelTime;
```
and then `useLightTravelTime = lightTravelTime.has_value()` and we don't need to specify it separately

That way it is impossible to set `useLightTravelTime` to true but then forget to set the observer or target